frontend/src/worksheets/TableListOfWorkSheets.js

```diff
@@ -1,19 +1,21 @@
 import React from 'react'
 import useWorkSheets from '../hooks/useWorkSheets'
+import FilterWorkSheets from './FilterWorkSheets'
 import { Link } from 'react-router-dom'
 import Button from '../Button'
 import { typeOfWork, status } from '../TranslationForWorkSheet'
 import LoadingScreen from '../LoadingScreen'
 
 export default function TableListOfWorkSheets() {
-  const { workSheets } = useWorkSheets()
+  const { workSheets, setStatus } = useWorkSheets()
 
   return (
     <>
-      <div className="d-flex justify-content-between p-1">
+      <div className="d-flex flex-row justify-content-around p-5">
         <Link to={`/worksheets/new`}>
           <Button text="Új munkalap létrehozása" moreClassName="w-auto p-1" />
         </Link>
+        <FilterWorkSheets onStatusChange={setStatus} />
```

Comment:
A status-t nem adod át, de a komponensben belül számítasz rá.

Reply:
Köszönöm, javítva.

Reply:
Ezért nem fut le a teszt:
src/worksheets/TableListOfWorkSheets.js
  Line 7:22:  'status' is defined but never used  no-unused-vars